proto/pulumi/resource.proto

```diff
@@ -148,6 +148,7 @@ message RegisterResourceRequest {
     optional bool retainOnDelete = 25;                          // if true the engine will not call the resource providers delete method for this resource.
     repeated Alias aliases = 26;                                // a list of additional aliases that should be considered the same.
     string deletedWith = 27;                                    // if set the engine will not call the resource providers delete method for this resource when specified resource is deleted.
+    repeated string replaceWith = 38;                           // if set the engine will replace this resource when any of the specified resources are replaced.
```

Comment:
```suggestion
    repeated string replace_with = 38;                           // if set the engine will replace this resource when any of the specified resources are replaced.
```
The camel case names are legacy, protobuf recommends you should use snake_case for property names. All new properties should use that.